pkg/webhook/validator.go

```diff
@@ -567,23 +576,25 @@ func ValidatePolicyAttestationsForAuthority(ctx context.Context, ref name.Refere
 			va, err := validAttestationsWithFulcio(ctx, ref, fulcioroot, rekorClient, authority.Keyless.Identities, remoteOpts...)
 			if err != nil {
 				logging.FromContext(ctx).Errorf("failed validAttestationsWithFulcio for authority %s with fulcio for %s: %v", name, ref.Name(), err)
-				return nil, fmt.Errorf("validate signatures with fulcio: %w", err)
```

Comment:
Similar to the API, I thought it prudent to avoid the term "fulcio" in error messages, especially since the API keyword is `keyless`.  I also tried to align this with the more detailed errors for the `key` path above.